dot/state/block.go

```diff
@@ -32,9 +32,23 @@ import (
 	"github.com/ChainSafe/gossamer/lib/database"
 )
 
+var blockPrefix = []byte("block")
+
 // BlockDB stores block's in an underlying Database
 type BlockDB struct {
-	Db database.Database
+	db database.Database
+}
+
+// Put appends `block` to the key and sets the key-value pair in the db
+func (db *BlockDB) Put(key, value []byte) error {
+	key = append(blockPrefix, key...)
+	return db.db.Put(key, value)
+}
+
+// Get appends `block` to the key and retrieves the value from the db
+func (db *BlockDB) Get(key []byte) ([]byte, error) {
+	key = append(blockPrefix, key...)
+	return db.db.Get(key)
```

Comment:
would be nice to avoid having (db.db) in multiple places as its redundant. perhaps changing the `func (db *BlockDB)` to `func (blockDB *BlockDB)` would produce less redundant expressions.

Would be nice to also do the same for `func (db *NetworkDB)` and `func (db *StorageDB)`

Reply:
ok done